HackIllinois/UI/UIViewExtensions.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  UIViewExtensions.swift
+//  HackIllinois
+//
+//  Created by Alex Drewno on 10/15/18.
+//  Copyright © 2018 HackIllinois. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+internal extension UIView {
+    func setConstraints(superView: UIView, topInset: CGFloat? = nil, bottomInset: CGFloat? = nil, widthInset: CGFloat? = nil, heightInset: CGFloat? = nil, leadingInset: CGFloat? = nil, trailingInset: CGFloat? = nil) {
```

Comment:
Generally we want to make sure the methods names read as close to a sentence as makes sense, so this method should be called `constrain(to view: UIView...)` as it reads better and is more expressive when used, ie: 

``` swift
contentView.constrain(to: cell, topInset: 0, trailingInset: 0, bottomInset: 0, leadingInset: 0)
```

Additionally, while it may seem weird, you should re-order the arguments to match the above. This is because it better matches apple's ordering as defined in `UIEdgeInsets`.